mwclient/client.py

```diff
@@ -612,6 +616,380 @@ def get_token(self, type, force=False, title=None):
 
         return self.tokens[type]
 
+    def create_user(self, username, password, **kwargs):
+        """
+         Creates user on the wiki with at least a username and a password.
+
+             >>> try:
+             ...     exists = site.create_user('SomeUser', 'Some password',
+             ...                               **my_extra_kwargs)
+             ... except mwclient.errors.CreateError as e:
+             ...     exists = e.code == 'userexists':
+             ...     if not exists:
+             ...         print('Can not create user: %s' % e)
+             ... if exists:
+             ...     user = site.get_user('SomeUser')
+
+         Args:
+             username (str): User name of the user to create
+             password (str): User password of the user to create
+             **kwargs (dict): Other required fields depending on your mediawiki conf
+                              eg: an email
+
+         Returns:
+             Bool (True) if user has beend created, else raise UserCreateError
+
+         Raises:
+             UserCreateError (mwclient.errors.UserCreateError): User can not be created
+                                                                for some reason
+             APIError (mwclient.errors.APIError): Other API errors
+        """
+
+        if 'createtoken' not in kwargs:
+            kwargs['createtoken'] = self.get_token('createaccount')
+        if 'retype' not in kwargs:
+            kwargs['retype'] = password
+        if 'continue' not in kwargs and 'createreturnurl' not in kwargs:
+            # should be great if API didn't require this...
+            kwargs['createreturnurl'] = '%s://%s' % (self.scheme, self.host)
+        info = self.post('createaccount', username=username, password=password, **kwargs)
+        if info['createaccount']['status'] == 'FAIL':
+            raise errors.UserCreateError(
+                info['createaccount'].get('messagecode', 'unknown'),
+                info['createaccount'].get('message', 'No reason...'),
```

Comment:
The defaults are a bit arbritrary. I suggest removing 'unknown' and 'No reason...' and just leave the default null value (`None`).

Reply:
done